packages/kafka-management-sdk/src/main/java/com/openshift/cloud/api/kas/DefaultApi.java

```
@@ -194,11 +194,11 @@ public String federateMetrics(String id) throws ApiException {
    * @param id The ID of record (required)
    * @param page Page index (optional)
    * @param size Number of items in each page (optional)
-   * @param instanceType The Kafka instance type to filter the results by (optional)
+   * @param UNKNOWN_PARAMETER_NAME  (optional)
    * @return a {@code CloudRegionList}
    * @throws ApiException if fails to make API call
    */
-  public CloudRegionList getCloudProviderRegions(String id, String page, String size, String instanceType) throws ApiException {
+  public CloudRegionList getCloudProviderRegions(String id, String page, String size,  UNKNOWN_PARAMETER_NAME) throws ApiException {
```

@jbriones - I still see instance type used as argument despite being deprecated. 
We are trying to remove deprecated fields in the Java SDK to fail fast for users to update those fields when upgrading SDK but if we are still using it as arg that causes issues